### ISSUE TITLE
keypad.Event differs from documentation

### ISSUE THREAD

**Issue by xs5871** [adafruit/circuitpython#5485]
### CircuitPython version

```python
Adafruit CircuitPython 7.0.0 on 2021-09-20; Raspberry Pi Pico with rp2040
```

### Code/REPL

```python
from keypad import Event
dir(Event)
```

### Behavior

```['__class__', '__name__', '__bases__', '__dict__', 'key_number', 'pressed', 'released']```

### Description

`keypad.Event` is missing implementations of members:
- `__hash__()`; Calling `hash()` on a `keypad.Event` yields the same integer, regardless of instance.
- `timestamp`; `AttributeError: type object 'Event' has no attribute 'timestamp'`.

**Comment by dhalbert** [comment]
Yes, timestamps were added posted 7.0.0. If you look at http://circuitpython.readthedocs.io/en/latest/, you'll see the doc for the "Absolute Newest" builds. If you look at http://circuitpython.readthedocs.io/en/, you'll see the doc for the latest stable release (which in this case does not include timestamps.

`hash()` was buggy, and I am fixing it. Thanks for spotting that.
